src/utils/registry.ts

```diff
@@ -203,7 +203,11 @@ export class Registry {
 
   constructor(packagePath: string) {
     const browsersJSON = JSON.parse(fs.readFileSync(path.join(packagePath, 'browsers.json'), 'utf8'));
-    this._descriptors = browsersJSON['browsers'];
+    this._descriptors = browsersJSON['browsers'].map((obj: any) => ({
+      name: obj.name,
```

Comment:
This way browser path stays the same and there is no way for us to tell if webkit-1444 in the cache is legacy mac14 build or the one that works for all newer OSes (if e.g. somebody puts it in a bug report). I can also imagine scenario where people upgrade their mac from 14 to something newer and use legacy build because it has matching version in the registry. Considering all that, I'd rather we put them in separate directories in the client side cache as well. WDYT?

Reply:
>  I can also imagine scenario where people upgrade their mac from 14 to something newer and use legacy build because it has matching version in the registry.

@yury-s Good point!